decred/tests/unit/examples/test_examples.py

```diff
@@ -0,0 +1,28 @@
+"""
+Copyright (c) 2019, The Decred developers
+"""
+
+import importlib.util
+import os
+from pathlib import Path
+import py_compile
+from tempfile import TemporaryDirectory
+
+import decred
+
+
+def test_compile():
+    exampleDir = Path(os.path.realpath(decred.__file__)).parent.parent / "examples"
```

Comment:
You could use `Path.resolve(__file__)` here. Using both `pathlib` and `os.path` in the same module is a bit ugly, even more so on the same line.